File: src/lightnap-ng/src/app/core/backend-api/dtos/users/request/search-user-claims-request-dto.ts
import { SearchClaimsRequestDto } from "./search-claims-request-dto";


/**
 * Represents a request to search for user claims with optional filtering and pagination.
 *
 * @extends SearchClaimsRequestDto
 *
 * @property {string} [userId] - The email address substring to search for.
 */
export interface SearchUserClaimsRequestDto extends SearchClaimsRequestDto {
   /**
     * The value to filter claims by.
     * @type {string}
     */
    userId?: string;
}

File: src/LightNap.Core/Users/Dto/Request/SearchUserClaimsRequestDto.cs
namespace LightNap.Core.Users.Dto.Request
{
    /// <summary>
    /// Represents a request to search user claims.
    /// </summary>
    public class SearchUserClaimsRequestDto : SearchClaimsRequestDto
    {
        /// <summary>
        /// Filter by user.
        /// </summary>
        public string? UserId { get; set; }
    }
}